service/routes.py

```diff
@@ -71,13 +70,9 @@ def create_inventory_records():
     check_content_type("application/json")
     inventory = Inventory()
     data = request.get_json()
-
-    if "condition" not in data or data["condition"] is None:
-        data["condition"] = "new"
-
     inventory.deserialize(data)
-    product = inventory.find((inventory.product_id, inventory.condition))
-    if product:
+    existing_inventory = Inventory.find((inventory.product_id, inventory.condition))
```

Comment:
I think to maintain consistency here also we can call find_from_request_json

Reply:
instead of 
```
inventory = Inventory()
data = request.get_json()
inventory.deserialize(data)
inventory.find((inventory.product_id, inventory.condition))
```

Reply:
So `Inventory.find((inventory.product_id, inventory.condition))` returns an object from the database. So it is already created. On the other hand, since we want to create a new record, we need the deserialized inventory object, which is not yet created. Thus did not call the function `find_from_request_json`.

Let me know if I am missing something.

Reply:
ohh got it..hmm 
so was just wondering did we create `find_from_request_json` function as we are repeatedly doing the same thing in other functions?

Reply:
Yes, there are some redundancy. Alright, we can think about it more and get on this with some other story.